lib/trollop.rb

```diff
@@ -220,7 +232,7 @@ def parse(cmdline = ARGV)
       vals[sym] = [] if opts.multi && !opts.default # multi arguments default to [], not nil
     end
 
-    resolve_default_short_options!
+    resolve_default_short_options! unless @settings[:no_default_short_opts]
```

Comment:
not thrilled with this name. Are there other option names that seem similar to this one?